src/edu/ucsb/cs56/projects/games/fish_animation/Fish.java

```diff
@@ -5,6 +5,7 @@
 import java.awt.geom.Line2D; 
 import java.awt.geom.PathIterator;
 import java.awt.geom.AffineTransform;
+import java.awt.geom.Ellipse2D;
```

Comment:
Is this used?  If it's not you should remove it.

Reply:
Deleted!